org.cosmic.Wam.json

```diff
@@ -0,0 +1,53 @@
+{
+    "app-id": "org.cosmic.Wam",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "23.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "cosmic-wam",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--share=network",
+        "--filesystem=/var/lib/flatpak:ro",
+        "--filesystem=xdg-data/flatpak:ro",
+        "--filesystem=xdg-data/applications:rw",
```

Comment:
Needs an exception https://docs.flathub.org/docs/for-app-authors/linter#exceptions

https://docs.flathub.org/docs/for-app-authors/linter#finish-args-unnecessary-xdg-data-access

Reply:
Ok, I opened new PR: https://github.com/flathub-infra/flatpak-builder-lint/pull/391
hope to not destroy anything haha :)

Reply:
Hey @bbhtt I can fix it by setting paths to ~/.local/share instead of xdg-data will it be better? 
In my app I'm specifically looking for this absolute paths instead of xdg-data env variable so maybe it should fix this issue?

Reply:
Ok, I changed it. Now linter does not shout out.